mobilecoind/strategies/drain-accounts.py

```diff
@@ -0,0 +1,156 @@
+                        help="Mobilecoind port")
+    parser.add_argument("--key-dir",
+                        type=str,
+                        help="Path to directory of account_keys")
+    parser.add_argument("--dest-key-dir",
+                        type=str,
+                        help="Path to directory of destination account_keys")
+    parser.add_argument("--max-seconds",
+                        type=int,
+                        default=40,
+                        help="Number of seconds to wait for a tx to clearn")
+    parser.add_argument("--fee",
+                        type=int,
+                        default=20,
+                        help="Amount less than the balance that we attempt to send")
+    parser.add_argument("--max-accounts",
+                        type=int,
+                        default=-1,
+                        help="Number of accounts to pull from the account keys / destination accounts keys folders")
+    return parser.parse_args()
+
+def read_file(path):
+    with open(path, "r") as file:
+        return file.read()
+
+def run_test(stub, amount, monitor_id, dest, max_seconds):
+    tx_stats = {}
+    sync_start = time.time()
+    wait_for_accounts_sync(stub, [monitor_id], 3)
+    print("Time to sync:", time.time() - sync_start)
+
+    resp = stub.GetBalance(
+        mobilecoind_api_pb2.GetBalanceRequest(monitor_id=monitor_id))
+    starting_balance = resp.balance
+    print("Starting balance prior to transfer:", starting_balance)
+    tx_resp = stub.SendPayment(
+        mobilecoind_api_pb2.SendPaymentRequest(
+            sender_monitor_id=monitor_id,
+            sender_subaddress=0,
+            outlay_list=[
+                mobilecoind_api_pb2.Outlay(
+                    value=amount,
+                    receiver=dest,
+                )
+            ],
+            fee=0,
+            tombstone=0,
+        ))
+
+    tx_stats[0] = {
+        'start': time.time(),
+        'time_delta': None,
+        'tombstone': tx_resp.sender_tx_receipt.tombstone,
+        'block_delta': None,
+        'status': TransferStatus.pending,
+        'receipt': tx_resp,
+    }
+    stats = poll(monitor_id, tx_stats, stub)
+    # FIXME: Move max seconds check inside polling
+    assert tx_stats[0]['time_delta'] < max_seconds, "Did not clear in time"
+    assert tx_stats[0]['status'] == TransferStatus.success, "Transfer did not succeed"
+    return stats
+
+# When Comparing filenames, make shorter file names compare less than longer filenames,
+# using this key function
+def filename_key(filename):
+    return (len(filename), filename);
+
+
+if __name__ == '__main__':
+    args = parse_args()
+    print(args)
+
+    stub = connect(args.mobilecoind_host, args.mobilecoind_port)
+    source_accounts = [
+        load_key_and_register("{}/{}".format(args.key_dir, k), stub)
+        for k in sorted(
+            filter(lambda x: x.endswith(".json"), os.listdir(args.key_dir)), key=filename_key)
```

Comment:
I think this filter/listdir can be replaced with [`glob`](https://docs.python.org/3/library/glob.html)
`glob.glob(os.path.join(args.key_dir, '*.json'))`

Reply:
can we do this in a follow up PR? i think this will require me modifying the requirements.txt and I don't know how to do that right now

Reply:
`glob` is a built-in module.

Reply:
ok will do